src/jllvm/materialization/ByteCodeCompileLayer.cpp

```diff
@@ -946,8 +947,28 @@ void codeGenBody(llvm::Function* function, const Code& code, const ClassFile& cl
             // TODO: DAdd
             // TODO: DCmpG
             // TODO: DCmpL
-            // TODO: DConst0
-            // TODO: DConst1
+            [&](OneOf<DConst0, DConst1, FConst0, FConst1, FConst2, IConstM1, IConst0, IConst1, IConst2, IConst3,
+                      IConst4, IConst5, LConst0, LConst1>)
+            {
+                auto* value = match(
+                    operation, [](...) -> llvm::Value* { llvm_unreachable("Invalid const operation"); },
+                    [&](DConst0) -> llvm::Value* { return llvm::ConstantFP::get(builder.getDoubleTy(), 0.0); },
```

Comment:
Ditto here, do we have to explicitly specify `-> llvm::Value*` for all of these? I'd prefer them gone if possible

Reply:
They are sadly necessary here

Reply:
Done